.golangci.yaml

```diff
@@ -23,7 +23,7 @@ linters-settings:
       - wrapperFunc
   gocyclo:
     min-complexity: 15
-  gomnd:
+  mnd:
```

Comment:
I'm confused why we never got a warning about this using regular lint. I only saw this in the github-runner-monitor lint failure, that was triggered a few days ago for no apparent reason.

Reply:
Probably because `mnd` was never turned on.

Also, it looks like `github-runner-monitor` was using `golangci/golangci-lint-action` instead of running `github.com/golangci/golangci-lint/cmd/golangci-lint@v1.64.6`, so probably something about being triggered from an action directly made it unhappy.